seacrowd/sea_datasets/indonesiannmt/indonesiannmt.py

```diff
@@ -0,0 +1,188 @@
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@misc{susanto2023replicable,
+      title={Replicable Benchmarking of Neural Machine Translation (NMT) on Low-Resource Local Languages in Indonesia}, 
+      author={Lucky Susanto and Ryandito Diandaru and Adila Krisnadhi and Ayu Purwarianti and Derry Wijaya},
+      year={2023},
+      eprint={2311.00998},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+_DATASETNAME = "indonesiannmt"
+
+_DESCRIPTION = """\
+This dataset is used on the paper "Replicable Benchmarking of Neural Machine Translation (NMT) on Low-Resource Local Languages in Indonesia". This repository contains two types of data:
+1. Monolingual (*.txt) [Indonesian, Javanese]
+2. Bilingual (*.tsv) [Indonesian-Javanese, Indonesian-Balinese, Indonesian-Minangkabau, Indonesian-Sundanese]
+Only the Bilingual dataset is available for this dataloader
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/Exqrch/IndonesianNMT"
+
+_LANGUAGES = ['ind', 'jav', 'ban', 'min', 'sun']  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value 
+
+_LOCAL = False
+
+_URLS = {
+    "ind_jav": "https://huggingface.co/datasets/Exqrch/IndonesianNMT/resolve/main/id-jv.tsv?download=true",
+    "ind_sun": "https://huggingface.co/datasets/Exqrch/IndonesianNMT/resolve/main/id-su.tsv?download=true",
+    "ind_ban": "https://huggingface.co/datasets/Exqrch/IndonesianNMT/resolve/main/id-ban.tsv?download=true",
+    "ind_min": "https://huggingface.co/datasets/Exqrch/IndonesianNMT/resolve/main/id-min.tsv?download=true",
+}
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]  
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+def seacrowd_config_constructor(modifier, schema, version):
+    return SEACrowdConfig(
+            name=f"indonesiannmt-{modifier}_{schema}",
+            version=version,
+            description=f"indonesiannmt-{modifier} {schema} schema",
+            schema=f"{schema}",
+            subset_id="indonesiannmt",
+        )
+
+class IndonesianNMT(datasets.GeneratorBasedBuilder):
+    """IndonesianNMT consists of 4 parallel datasets and 2 monolingual datasets, 
+    all obtained synthetically from either gpt-3.5-turbo or text-davinci-003"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = (
+        [SEACrowdConfig(
+            name=f"indonesiannmt_source",
+            version=_SOURCE_VERSION,
+            description=f"indonesiannmt source schema",
+            schema=f"source",
+            subset_id="indonesiannmt",
+        )] + 
+        [SEACrowdConfig(
+            name=f"indonesiannmt_seacrowd_t2t",
```

Comment:
There's no need for `indonesiannmt_seacrowd_t2t` if it only loads the Javanese data. We can remove this subset.